Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/Overrides/PrefabOverridePublicHandler.cpp

```diff
@@ -56,9 +56,10 @@ namespace AzToolsFramework
             AZStd::pair<AZ::Dom::Path, LinkId> pathAndLinkIdPair = GetPathAndLinkIdFromFocusedPrefab(entityId);
             if (!pathAndLinkIdPair.first.IsEmpty() && pathAndLinkIdPair.second != InvalidLinkId)
             {
-                if (!relativePathFromEntity.empty())
+                AZ::Dom::Path relativeDomPathFromEntity(relativePathFromEntity);
```

Comment:
nit: Can avoid explicit declaration here by changing line 62 to AZ::Dom::Path(relativeDomPathFromEntity) . Fyi, this is what I added in https://github.com/o3de/o3de/pull/14784